<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <comment>
     MIDI Virtual Piano Keyboard
     Copyright (C) 2008-2019, Pedro Lopez-Cabanillas

     This program is free software; you can redistribute it and/or modify
     it under the terms of the GNU General Public License as published by
     the Free Software Foundation; either version 3 of the License, or
     (at your option) any later version.

     This program is distributed in the hope that it will be useful,
     but WITHOUT ANY WARRANTY; without even the implied warranty of
     MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
     GNU General Public License for more details.

     You should have received a copy of the GNU General Public License along
     with this program; If not, see http://www.gnu.org/licenses/
 </comment>
 <class>SonivoxSettingsDialog</class>
 <widget class="QDialog" name="SonivoxSettingsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>304</width>
    <height>280</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Sonivox EAS Synth</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>:/icon.png</normaloff>:/icon.png</iconset>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout">
     <item row="2" column="2">
      <widget class="QLabel" name="lblChorus">
       <property name="text">
        <string>Chorus</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>8</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="3" column="2">
      <widget class="QDial" name="dial_Chorus">
       <property name="maximum">
        <number>32765</number>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="lblReverb">
       <property name="text">
        <string>Reverb</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="0" column="0" colspan="3">
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="lblBufferTime">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Minimum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Buffer Time:</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="spnTime">
         <property name="suffix">
          <string> ms</string>
         </property>
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="maximum">
          <number>9999</number>
         </property>
         <property name="value">
          <number>60</number>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="4" column="0">
      <widget class="QComboBox" name="combo_Reverb"/>
     </item>
     <item row="4" column="2">
      <widget class="QComboBox" name="combo_Chorus"/>
     </item>
     <item row="3" column="0">
      <widget class="QDial" name="dial_Reverb">
       <property name="maximum">
        <number>32765</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok|QDialogButtonBox::RestoreDefaults</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>spnTime</tabstop>
  <tabstop>dial_Reverb</tabstop>
  <tabstop>dial_Chorus</tabstop>
  <tabstop>combo_Reverb</tabstop>
  <tabstop>combo_Chorus</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>SonivoxSettingsDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>145</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>148</x>
     <y>288</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>SonivoxSettingsDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>239</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>251</x>
     <y>289</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
